.travis.yml

```diff
@@ -29,20 +31,20 @@ script:
 - make -j5 && make install
 - popd
 
-#clone and build mesa : dependency
+# #clone and build mesa : dependency
 - git clone https://github.com/intel/external-mesa /tmp/mesa
-- pushd /tmp/mesa
-- ./autogen.sh --prefix=$WLD --with-platforms=surfaceless --disable-dri3 --enable-shared-glapi --disable-glx --disable-glx-tls --disable-gbm --without-gallium-drivers --with-dri-drivers=i965 --with-vulkan-drivers=intel
-- make CFLAGS=-DSYS_memfd_create=319 CPPFLAGS=--std=c11 -j5 && make install
-- popd
+# - pushd /tmp/mesa
+# - ./autogen.sh --prefix=$WLD --with-platforms=surfaceless --disable-dri3 --enable-shared-glapi --disable-glx --disable-glx-tls --disable-gbm --without-gallium-drivers --with-dri-drivers=i965 --with-vulkan-drivers=intel
+# - make CFLAGS=-DSYS_memfd_create=319 CPPFLAGS=--std=c11 -j5 && make install
+# - popd
 
-#clone and build minigbm : dependency
-- git clone https://github.com/intel/minigbm /tmp/minigbm
-- pushd /tmp/minigbm
-- export DRV_I915=1
-- make CFLAGS=-DDRV_I915=1 -j5
-- make install DESTDIR=$WLD LIBDIR=lib/
-- popd
+# #clone and build minigbm : dependency
+# - git clone https://github.com/intel/minigbm /tmp/minigbm
```

Comment:
Why is this commented ? I think we want to have two clean builds one using Minigbm and one using GBM